toolchain/check/testdata/builtin_conversions/no_prelude/convert_class_type_to_generic_facet_value.carbon

```diff
@@ -433,8 +433,6 @@ fn G() {
 // CHECK:STDOUT:   %Generic.type.loc17_45.2 => constants.%Generic.type.769
 // CHECK:STDOUT:   %U.loc17_32.2 => constants.%Generic.facet
 // CHECK:STDOUT:   %U.patt.loc17_32.2 => constants.%Generic.facet
-// CHECK:STDOUT:
-// CHECK:STDOUT: !definition:
```

Comment:
Are these changes expected? I can sort of see how this PR could lead to some specifics gaining definitions, but it's harder to understand why some lose their definitions (or am I misunderstanding what `!definition:` signifies?)

Reply:
This is happening for specifics whose argument list contains an error. In that case we don't reach the `EvalConstantInst` call, so we don't attempt to create a definition any more.